build/images/scripts/install_whereabouts_config

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -u -e
+
+# Inspired by: https://github.com/intel/multus-cni/blob/83556f49bd6706a885eda847210b542669279cd0/images/entrypoint.sh#L161-L222
```

Comment:
nit: could you update the link to point to the new multus repo?

additionally, I don't see an Intel 2018 copyright in the original file, so maybe we do not need it here either?

Reply:
removed copyright info.